cita-cli/src/cli.rs

```diff
@@ -19,7 +19,7 @@ use cita_tool::client::system_contract::{
 use cita_tool::{
     decode_input, decode_logs, decode_params, encode, encode_input, encode_params, parse_url,
     pubkey_to_address, remove_0x, KeyPair, ParamsValue, PrivateKey, ProtoMessage, PubKey,
-    ResponseValue, UnverifiedTransaction,
+    ResponseValue, TransactionOption, UnverifiedTransaction,
```

Comment:
`struct` name should be `TransactionOptions` or `TxOptions`

`variable` name maybe `tx_opts` or `tx_options`